components/dashboard/src/components/DropDown.tsx

```diff
@@ -35,14 +36,20 @@ function DropDown(props: DropDownProps) {
             },
         };
     });
-    const font =
-        "text-gray-400 dark:text-gray-500 text-sm leading-1 group hover:text-gray-600 dark:hover:text-gray-400 transition ease-in-out";
+    const defaultFont = "text-gray-400 dark:text-gray-500 hover:text-gray-600 dark:hover:text-gray-400 ";
+    const asLinkFont = "text-blue-500 dark:text-blue-400 hover:text-blue-600 dark:hover:text-blue-500";
+    const asLinkArrowBorder =
+        "border-blue-500 dark:border-blue-400 group-hover:border-blue-600 dark:group-hover:border-blue-500";
     return (
-        <ContextMenu menuEntries={enhancedEntries} classes={`${props.contextMenuWidth} right-0`}>
-            <span className={`py-2 cursor-pointer ${font}`}>
+        <ContextMenu menuEntries={enhancedEntries} customClasses={`${props.customClasses} right-0`}>
+            <span
+                className={`py-2 cursor-pointer text-sm leading-1 group ${
+                    props.renderAsLink ? asLinkFont : defaultFont
+                } transition ease-in-out`}
+            >
```

Comment:
**thought**: This is interesting! Is this only about the color change? If yes, it could be fine to leave this without the color change to match the rest of the link dropdowns but it's fine to also leave this as is for for now. Your call.

| BEFORE | AFTER |
|-|-|
| <img width="549" alt="link-before" src="https://user-images.githubusercontent.com/120486/170995934-c4ea3070-bcc6-4817-bb59-31051e1f9b66.png"> | <img width="549" alt="link-after" src="https://user-images.githubusercontent.com/120486/170995948-3f2b034d-e353-4da8-bd39-1bffabbe7af7.png"> |

Reply:
Yes, all these were only for the color change. 😅 But I think it's worth it, because with @jldec we found out that the grey currency switcher was very hard to notice when not link-color.

I think this is a nice improvement. ✨

Reply:
DEAL. Let's keep it for now. Maybe we could update all link dropdowns to become blue in the long run. Or the other way around.